config/schemas/config.yaml

```
@@ -6582,6 +6579,17 @@ properties:
     title: Issuers Config
     description: Configure issuers for cert-manager.
     type: object
+
+    # Need to either define your own issuers or enable letsencrypt
+    anyOf:
+      - properties:
+          letsencrypt:
+            properties:
+              enabled:
+                type: boolean
+                const: true
+      - required:
+          - extraIssuers
```

We can enforce that minimum one extraIssuer is defined by adding the property in the `anyOf` block instead:
```suggestion
    anyOf:
      - properties:
          letsencrypt:
            properties:
              enabled:
                type: boolean
                const: true
      - properties:
          letsencrypt:
            properties:
              enabled:
                type: boolean
                const: false
          extraIssuers:
            type: array
            minItems: 1
        required:
          - extraIssuers
```